### DISCUSSION TITLE
Groesten Haus High round record

### DISCUSSION THREAD

**Discussion by ToonLink0001** [nzp-team/nzportable#693] [Records and Speedruns]
Got downed because of my mouse unpluging, could have done better in my opinion
![Nazi Zombies_ Portable 3_25_2024 10_19_03 PM](https://github.com/nzp-team/nzportable/assets/164558069/54ef778d-6d02-4442-ba03-1c79e527241d)

**Comment by ElFish09** [comment]
Did this like a month ago when the map came out lol
![imagen_2024-03-26_164320524](https://github.com/nzp-team/nzportable/assets/107020918/d4d2ba5d-5b63-447c-b8d7-298c2ea7637b)

**Comment by DGxShAdOwZz** [comment]
![image_2024-04-28_110442499](https://github.com/nzp-team/nzportable/assets/128944577/e9b6a9e8-cbc5-4b6d-a270-da967ec33a2b)
back after a while but yeah could do better when i get back into it

**Comment by Kingjaxgaming420** [comment]
![Screenshot (3)](https://github.com/user-attachments/assets/e698588e-0738-40a5-a66a-9002f6734c3e)

**Reply by Kingjaxgaming420** [reply]
https://youtu.be/QYc6jO7Co4M?si=Vb0soKu2vc2Ekh3y
